src/components/ArticleActions.vue

```diff
@@ -9,23 +9,23 @@
     &nbsp;&nbsp;
     <button
       class="btn btn-outline-danger btn-sm"
-      @click="deleteArticle(article.slug)">
+      @click="deleteArticle()">
```

Comment:
We can even omit the parentheses here. We should rather use something like `@click="deleteArticle"`.